### PR TITLE
Add env overrides in hello world agent package

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

This PR adds env overrides in the hello world agent package

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

### CODE REVIEW

File: packages/valory/agents/hello_world/aea-config.yaml [hunk 1]
```diff
@@ -52,10 +52,21 @@ dependencies:
...
+      tendermint_url: ${TENDERMINT_URL:str:http://localhost:26657}
+      tendermint_com_url: ${TENDERMINT_COM_URL:str:http://localhost:8080}
+---
 public_id: valory/abci:0.1.0
 type: connection
 config:
   target_skill_id: valory/hello_world_abci:0.1.0
+  host: ${ABCI_HOST:str:localhost}
+  port: ${ABCI_PORT:int:26658}
+  use_tendermint: ${ABCI_USE_TENDERMINT:bool:false}
```
Comment (angrybayblade) [inline packages/valory/agents/hello_world/aea-config.yaml:69]: It seems the agent package was missing the env overridable

File: packages/valory/agents/hello_world/aea-config.yaml [hunk 2]
```diff
@@ -52,10 +52,21 @@ dependencies:
   open-aea-ledger-ethereum: {}
 default_connection: null
 ---
+public_id: valory/hello_world_abci:0.1.0
+type: skill
+models:
+  params:
+    args:
+      tendermint_url: ${TENDERMINT_URL:str:http://localhost:26657}
+      tendermint_com_url: ${TENDERMINT_COM_URL:str:http://localhost:8080}
```
Comment (angrybayblade) [inline packages/valory/agents/hello_world/aea-config.yaml:61]: Same

File: docs/autonomy.md [hunk 3]
```diff
@@ -75,7 +75,7 @@ These keys can be used for local deployments if you're using the default hardhat
 
 ```bash
 # create a docker deployment
-$ autonomy deploy build deployment valory/oracle_hardhat:bafybeibhv2ziivbnj3sgfzjjtvqbsvt3fvra4texc3lx4snqxo6lbmq2le keys.json
+$ autonomy deploy build deployment valory/oracle_hardhat:bafybeigw2hpuir72fca6oxfvh4kmy3rrllzivchw63tdi5ylkzvvssybvm keys.json
```
Comment (angrybayblade) [inline docs/autonomy.md:78]: @dvilelaf @jmoreira-valory Can you guys check if this works?

Comment (dvilelaf) [reply docs/autonomy.md:78]: It's working on my side

File: packages/valory/agents/hello_world/aea-config.yaml (hunk 2 above)
Comment (dvilelaf) [reply packages/valory/agents/hello_world/aea-config.yaml:61]: why defined on the agent and not on the service?

**Review by dvilelaf — DISMISSED** [review]

File: docs/autonomy.md (hunk 3 above)
Comment (dvilelaf) [reply docs/autonomy.md:78]: Why has oracle_hardhat changed its hash here?

Comment (angrybayblade) [reply docs/autonomy.md:78]: Oh I thought this was hello world service

File: packages/valory/agents/hello_world/aea-config.yaml (hunk 2 above)
Comment (angrybayblade) [reply packages/valory/agents/hello_world/aea-config.yaml:61]: So these are environment overrides which mean these variables will be picked from the env at the runtime. These env vars will be defined by the service and used by the agent that's why they're defined on agent config

**Review by dvilelaf — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**